ua-converter/ua_convert.py

```diff
@@ -64,35 +114,96 @@ def process_item(item):
         "device_brand": ua_result.device.brand if ua_result.device else None,
         "browser": ua_result.user_agent.family if ua_result.user_agent else None,
         "browser_version": browser_version,
-        "browser_version_major_minor": browser_version_major_minor,
+        "version": browser_version_major_minor,
```

Comment:
I called it `browser_version_major_minor` on purpose. Since `version` is too ambitious. After all, there is a browser version as well as an OS version. So don't call it `version` :)

Reply:
Yes yes, that's the rebase I meant was pending. I was trying to keep "backwards compatibility" and then you submitted the PR where you made the change effective. So this will change when I do the rebase.